src/usr/local/www/system_usermanager_settings.php

```diff
@@ -199,6 +205,15 @@
 	$auth_servers
 ));
 
+$section->addInput(new Form_Input(
+        'auth_refresh_time',
+        'Auth Refresh Time',
+        'number',
+        $pconfig['auth_refresh_time'],
+        ['min' => 0, 'max' => 3600]
+))->setHelp('Time in seconds to cache authentication results. The default is 30 seconds, maximum 3600 (one hour). '.
```

Comment:
The time restrictions must also be checked with input validation of the POST values. The check on the line above is handled client-side and cannot be trusted alone.